### PR TITLE
🚧 Tab List Block - Code Review

### PR DESCRIPTION
**Pull request by SteveJonesDev**
This PR has code review changes and TODO's. Please review the changes and address all TODO comments. Once each TODO is fixed its comment can be removed. Please check out this PR branch and make changes to it. Set me as the reviewer once the changes are made.

FYI: my code changes are mostly untested so please test thoroughly.

### CODE REVIEW

File: src/js/global.js [hunk 1]
```diff
@@ -321,3 +321,145 @@ jQuery(document).ready(function () {
 		});
 	}
 });
+
+// tab list block
+jQuery(document).ready(function () {
```
Comment (SteveJonesDev) [inline src/js/global.js:326]: @tronsymphony, this needs to be a separate javascript file that gets loaded via the block.js file. Also, this script doesn't work when multiple of the same block are added to the page.

Comment (SteveJonesDev) [reply src/js/global.js:326]: If registered properly in the block.json file the script should load on both the front end and back end.

File: src/scss/blocks/_tab-list-block.scss [hunk 2]
```diff
@@ -1,12 +1,12 @@
-.block_tab-list-block {
+.tab-list-block {
     background-color: #120D2D;
 
-    .container {
+    .tab-list-block_container {
```
Comment (SteveJonesDev) [inline src/scss/blocks/_tab-list-block.scss:4]: Use: `&_container {`

File: src/scss/blocks/_tab-list-block.scss [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
         max-width: 1121px;
         margin: 0 auto;
     }
 
-    .tabs {
+    .tab-list-block_container_tabs {
```
Comment (SteveJonesDev) [inline src/scss/blocks/_tab-list-block.scss:9]: @tronsymphony Use: `&_container_tabs`

File: src/scss/blocks/_tab-list-block.scss [hunk 4]
```diff
@@ -21,7 +21,7 @@
         margin: 40px 0 0;
     }
 
-    .tabs-container {
+    .tab-list-block_container_tabs-container {
```
Comment (SteveJonesDev) [inline src/scss/blocks/_tab-list-block.scss:24]: @tronsymphony Use: `&_container_tabs-container`

File: partials/blocks/tab-list-block/template.php [hunk 5]
```diff
@@ -33,237 +33,94 @@
...
 $class_name = apply_filters( 'loader_block_class', $class_name, $block, $post_id );
-
 ?>
 
 <section id="<?php echo esc_attr( $block_id ); ?>" class="<?php echo esc_attr( $class_name ); ?>">
-	<div class="container">
-		<div class="tabs">
-				<div role="tablist" aria-labelledby="tablist-1" class="manual">
+	<div class="tab-list-block_container">
+		<div class="tab-list-block_container_tabs tabs">
```
Comment (SteveJonesDev) [inline partials/blocks/tab-list-block/template.php:40]: @tronsymphony there `tabs` class is still here.

**Comment by SteveJonesDev** [comment]
@tronsymphony, I added some comments to this PR. The big issue is the loading of the JavaScript in the block.json file so scripts work in the front end as well as the back end. Also, you'll need to merge the develop branch into this one and resolve merge conflicts.

File: src/js/global.js (hunk 1 above)
Comment (tronsymphony) [reply src/js/global.js:359]: @SteveJonesDev Please advise on how to do this

**Pull request merged**